### PR TITLE
Deprecate type1 transactions

### PR DESCRIPTION
**Pull request by GregTheGreek**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by axic** [comment]
Isn't this really just a recommendation for tooling not to use 2930-style transactions?

If so, doesn't this belong under the `Informational` category?  Or does it really need an EIP?  Another option I could see is if we start issuing "Deprecation" notices under the `Core` category, which really bare no real action, just being a deprecation notice.

However if you want to reject such transactions altogether, that is a `Core` change.

**Comment by GregTheGreek** [comment]
> If so, doesn't this belong under the `Informational` category?
This probably makes more sense, I was following suit on some previous EIPs i saw.

> Another option I could see is if we start issuing "Deprecation" notices under the Core category, which really bare no real action, just being a deprecation notice.
I think this also would be a great addition to make! I fear with `TransactionTypes` in general we're probably going to end up deprecating old ones as we move forward, so this could be a great first step.

> However if you want to reject such transactions altogether, that is a Core change.
I would __love__ to, but that would be unnecessarily chaotic and complicated on the protocol layer.

**Comment by axic** [comment]
> > However if you want to reject such transactions altogether, that is a Core change.
>
> I would love to, but that would be unnecessarily chaotic and complicated on the protocol layer.

How would that be chaotic on the protocol layer?  I think it is pretty simple on the protocol layer.

It can however be chaotic on the wallet/dapp/ecosystem side ensuring that everyone is aware of the change.  The biggest hurdle as always is hardware wallets, as those, understandably, are the slowest in adoption/change.

**Comment by GregTheGreek** [comment]
> It can however be chaotic on the wallet/dapp/ecosystem side ensuring that everyone is aware of the change. The biggest hurdle as always is hardware wallets, as those, understandably, are the slowest in adoption/change.

I guess i was thinking they would upgrade type1->type2 but simply rejecting them would be easier...

**Comment by MicahZoltu** [comment]
A Core EIP would be one that actually removes it from the protocol, like makes blocks with 2930 transactions invalid.

Personally, I think it is more critical to figure out a way to EOL legacy transactions than 2930 transactions.  Legacy transactions don't follow the new 2718 transaction format and generally have a bunch of special case handling.

File: EIPS/eip-3709.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+review-period-end: TODO
+type: Standards Track
+category: Interface
+created: 2021-08-07
+requires: 1559
+---
+
+## Simple Summary
+
+Deprecates usage of `TransactionType` 1 on wallets, and providers, upgrading all type 1 transactions to a type 2 transaction.
```
Comment (lightclient) [inline EIPS/eip-3709.md:16]:
```suggestion
Deprecates usage of [EIP-2718](./eip-2718.md) `TransactionType` 1 on wallets, and providers, upgrading all type 1 transactions to a type 2 transaction.
```

File: EIPS/eip-3709.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+--
```
Comment (lightclient) [inline EIPS/eip-3709.md:1]:
```suggestion
---
```

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-3709.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+eip: 3709
+title: Remove Support for Type-1 Transactions
```
Comment (lightclient) [inline EIPS/eip-3709.md:3]:
```suggestion
title: Remove Support for Type 1 Transactions
```

File: EIPS/eip-3709.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+author: Gregory Markou (@GregTheGreek)
+discussions-to: https://ethereum-magicians.org/t/eip-3709-deprecate-type-1-transactions/6810
+status: Draft
+review-period-end: TODO
```
Comment (lightclient) [inline EIPS/eip-3709.md:7]:
```suggestion
```

File: EIPS/eip-3709.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+type: Standards Track
+category: Interface
+created: 2021-08-07
+requires: 1559
+---
+
+## Simple Summary
+
+Deprecates usage of [EIP-2718](./eip-2718.md) `TransactionType` 1 on wallets, and providers, upgrading all type 1 transactions to a type 2 transaction.
```
Comment (lightclient) [inline EIPS/eip-3709.md:16]:
```suggestion
Deprecates usage of [EIP-2718](./eip-2718.md) `TransactionType` 1 in wallets and providers, upgrading all type 1 transactions to a type 2 transaction.
```

File: EIPS/eip-3709.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+
+## Specification
+
+For wallets and providers, if a user submits a transaction for signing with where `TransactionType == 0x1`, the developer should upgrade the transaction to meet the criteria of transaction of type 2.
+
+The following fields need to be changed, or ammended:
```
Comment (lightclient) [inline EIPS/eip-3709.md:30]:
```suggestion
The following fields need to be changed, or amended:
```

**Review by lightclient — APPROVED** [review]

**Pull request merged**